site/en/docs/devtools/css/reference/index.md

```diff
@@ -121,6 +124,48 @@ If your browser supports the `lab()` function, the element is green, otherwise i
 **Note**: At the time of writing, only Safari [supports the CIELAB color space](https://caniuse.com/?search=lab).
 {% endAside %}
 
+### View cascade layers {: #cascade-layers}
```

Comment:
Mov eit to lower part of the document as this is not as important and common feature than others.